### PR TITLE
[Merged by Bors] - chore(SetTheory/Cardinal/Ordinal): remove 200 line porting note

### PR DESCRIPTION
**Pull request by vihdzp**
As the porting note itself explains, this code can't be used in Lean 4.

I'd love to discuss what can be done to add this functionality back. But for the moment, this is taking up a lot of space in a file dangerously close to the 1.5k line limit.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/SetTheory/Cardinal/Ordinal.lean [hunk 1]
```diff
@@ -1225,209 +1225,6 @@ theorem extend_function_of_lt {α β : Type*} {s : Set α} (f : s ↪ β) (hs :
     rwa [mk_compl_of_infinite s hs, mk_compl_of_infinite]
     rwa [← lift_lt, mk_range_eq_of_injective f.injective, ← h, lift_lt]
 
-
--- Porting note: we no longer express literals as `bit0` and `bit1` in Lean 4, so we can't use this
--- section Bit
-
--- /-!
--- This section proves inequalities for `bit0` and `bit1`, enabling `simp` to solve inequalities
```
Comment (b-mehta) [inline Mathlib/SetTheory/Cardinal/Ordinal.lean:1233]: The claim here is that this is for numeral cardinals, but the `norm_cast, norm_num` proof seems to work for them, so maybe that part should be kept?

Comment (vihdzp) [reply Mathlib/SetTheory/Cardinal/Ordinal.lean:1233]: I think this was supposed to work for `aleph` and such as well. Goals between numeric cardinals can be solved through simply `norm_cast` - I've added back `one_lt_two` as an example of this.

Comment (b-mehta) [reply Mathlib/SetTheory/Cardinal/Ordinal.lean:1233]: Maybe, but did anyone really write things like `bit0 (aleph _)`?

Comment (vihdzp) [reply Mathlib/SetTheory/Cardinal/Ordinal.lean:1233]: Yeah, no. In fact, I'm pretty sure this whole section went unused.

**Review by jcommelin — COMMENTED** [review]
Thanks :tada:

bors merge

**Pull request closed without merging**